### PR TITLE
Enable complexity checking in complexity checking docs example

### PR DESCRIPTION
**Pull request by anoadragon453**
@youphyun noticed in https://github.com/matrix-org/synapse/issues/8428#issuecomment-1040223558 that the recommended config for complexity checking on single board computers (SBCs) did not actually enable complexity checking.

This PR uncomments the `enabled: true` line in order to actually enable it.

Complexity checking is disabled by default: https://github.com/matrix-org/synapse/blob/119edf51eb3e4f5ed5139dc370f5d7aed46edc1c/synapse/config/server.py#L231-L232

### CODE REVIEW

File: docs/other/running_synapse_on_single_board_computers.md [hunk 1]
```diff
@@ -38,8 +38,7 @@ use_presence: false
 # joins a new remote room. If it is above the complexity limit, the server will
 # disallow joining, or will instantly leave.
```
Comment (clokep) [inline docs/other/running_synapse_on_single_board_computers.md:39]: The above sounds like you may or may not want to enable it. I think we should reword this to be clearer:

> Disable room "complexity" checking before a user joins a new remote room.
> If it is above the complexity limit, the server will disallow joining, or will
> instantly leave.

Comment (clokep) [reply docs/other/running_synapse_on_single_board_computers.md:39]: Ahhh, well that's confusing that it is copied from the sample config when bits and pieces of the values are modified. That's probably why it doesn't make sense.

I think it would be clearer, yes. We're telling people to paste this into their config, so it should describe what that change actually does IMO?

Comment (anoadragon453) [reply docs/other/running_synapse_on_single_board_computers.md]: I've made the comments more relevant in c30b24717. Hopefully that's better?

File: docs/other/running_synapse_on_single_board_computers.md [hunk 2]
```diff
@@ -31,28 +31,29 @@ Anything that requires modifying the device list [#7721](https://github.com/matr
...
   name: psycopg2
   args:
     user: matrix-synapse
-    # Generate a long, secure one with a password manager
+    # Generate a long, secure password using a password manager
     password: hunter2
     database: matrix-synapse
     host: localhost
-    cp_min: 5
-    cp_max: 10
```
Comment (clokep) [inline docs/other/running_synapse_on_single_board_computers.md:55]: Why this change?

**Review by clokep — APPROVED** [review]

File: docs/other/running_synapse_on_single_board_computers.md (hunk 2 above)
Comment (anoadragon453) [reply docs/other/running_synapse_on_single_board_computers.md:55]: Those are the default values and don't seem to be relevant to SBCs. Rather, I think they were just copy-pasted over.

Comment (clokep) [reply docs/other/running_synapse_on_single_board_computers.md:55]: Makes sense. Thanks!
